.travis.yml

```diff
@@ -101,210 +76,197 @@ _helpers:
-    services:
-    - docker
-    script:
-    - pip install -r requirements/cython.txt
-    - pip install -r requirements/ci.txt  # to compile *.c files by Cython
-    - make cythonize
-    - ./tools/run_docker.sh "aiohttp"
-    deploy:
-      <<: *deploy_step
-
-    # Build and deploy MacOS binary wheels for each OSX+Python combo possible
-    # OS X 10.10, Python 3.5
+  # Build and deploy MacOS binary wheels for each OSX + Python combo possible
   - <<: *osx_pypi_deploy_base_1010
-    python: 3.5
-    env:
-    - *env_os1010_msg
-    - *env_py35
-    - *env_pyenv
-    - *env_path
-    # OS X 10.10, Python 3.6
+    name: binary wheel python 3.5 on OS X 10.10
+    env: *osx_env_py35
   - <<: *osx_pypi_deploy_base_1010
-    env:
-    - *env_os1010_msg
-    - *env_py36
-    - *env_pyenv
-    - *env_path
-    # OS X 10.10, Python 3.7
+    name: binary wheel python 3.6 on OS X 10.10
+    env: *osx_env_py36
   - <<: *osx_pypi_deploy_base_1010
-    env:
-    - *env_os1010_msg
-    - *env_py37
-    - *env_pyenv
-    - *env_path
-    # OS X 10.11, Python 3.5
+    name: binary wheel python 3.7 on OS X 10.10
+    env: *osx_env_py37
   - <<: *osx_pypi_deploy_base_1011
-    python: 3.5
-    env:
-    - *env_os1011_msg
-    - *env_py35
-    - *env_pyenv
-    - *env_path
-    # OS X 10.11, Python 3.6
+    name: binary wheel python 3.5 on OS X 10.11
+    env: *osx_env_py35
   - <<: *osx_pypi_deploy_base_1011
-    env:
-    - *env_os1011_msg
-    - *env_py36
-    - *env_pyenv
-    - *env_path
-    # OS X 10.11, Python 3.7
+    name: binary wheel python 3.6 on OS X 10.11
+    env: *osx_env_py36
   - <<: *osx_pypi_deploy_base_1011
-    env:
-    - *env_os1011_msg
-    - *env_py37
-    - *env_pyenv
-    - *env_path
-    # OS X 10.12, Python 3.5
+    name: binary wheel python 3.7 on OS X 10.11
+    env: *osx_env_py37
   - <<: *osx_pypi_deploy_base_1012
-    python: 3.5
-    env:
-    - *env_os1012_msg
-    - *env_py35
-    - *env_pyenv
-    - *env_path
-    # OS X 10.12, Python 3.6
+    name: binary wheel python 3.5 on OS X 10.12
+    env: *osx_env_py35
   - <<: *osx_pypi_deploy_base_1012
-    env:
-    - *env_os1012_msg
-    - *env_py36
-    - *env_pyenv
-    - *env_path
-    # OS X 10.12, Python 3.7
+    name: binary wheel python 3.6 on OS X 10.12
+    env: *osx_env_py36
   - <<: *osx_pypi_deploy_base_1012
-    env:
-    - *env_os1012_msg
-    - *env_py37
-    - *env_pyenv
-    - *env_path
+    name: binary wheel python 3.7 on OS X 10.12
+    env: *osx_env_py37
+
+  # Build and deploy manylinux1 binary wheels and source distribution
+  - <<: *linux_pypi_deploy_base
+    name: Build and deploy source distribution and manylinux1 binary wheels
+    services:
+    - docker
+    script:
+    - ./tools/run_docker.sh "aiohttp"
 
 stages:
 - *doc_stage_name
 - test
 - name: *stage_test_osx_name
   if: type IN (api, cron)
 - name: *deploy_stage_name
-  # This will prevent deploy unless it's a tagged commit:
-  if: tag IS present
+  if: type IN (api, cron)
```

Comment:
This is wrong. Tag is absolutely necessary here. We don't publish intermediate commits to PyPI.

Reply:
Yes, I had come to the same conclusion and made the [same change for frozenlist](https://github.com/mjpieters/frozenlist/commit/b92ae33b292e3caf797e6a2ea0315f2cb16b09a9).